pmm_qa/pdpgsql_pgsm_setup.yml

```diff
@@ -49,7 +50,6 @@
   - name: Install pmm2-client on the pdpgsql_pgsm_container
     shell: "{{ item }}"
     with_items:
-      - docker exec {{ pdpgsql_pgsm_container }} wget https://raw.githubusercontent.com/Percona-Lab/qa-integration/{{ pmm_qa_branch }}/pmm_qa/pmm3-client-setup.sh
```

Comment:
pmm_qa_branch/qa_integration_branch can be used here instead as its an input parameter . This was added by @puneet0191 i think, to ensure we can choose different branch client setup scripts during testing if requried. I have also found it useful.

Reply:
so this change seems fine to me